prowler/providers/aws/services/rds/rds_instance_deletion_protection/rds_instance_deletion_protection.py

```diff
@@ -10,16 +10,25 @@ def execute(self):
             report.region = db_instance.region
             report.resource_id = db_instance.id
             report.resource_tags = db_instance.tags
-            if db_instance.deletion_protection:
-                report.status = "PASS"
-                report.status_extended = (
-                    f"RDS Instance {db_instance.id} deletion protection is enabled."
-                )
+            # Check if is member of a cluster
+            if db_instance.cluster_id:
+                for db_cluster in rds_client.db_clusters:
+                    if db_cluster.id == db_instance.cluster_id:
```

Comment:
Do you need this? You can access directly the cluster using the `cluster_id` right?

Reply:
No, because rds_client.db_clusters is a list

Reply:
Done